common/model/events.js

```diff
@@ -111,21 +111,24 @@ export type EventPromo = {|
   id: string,
   title: ?string,
   url: string,
-  start: ?DateTimeRange,
-  end: ?DateTimeRange,
+  start: ?Date,
+  end: ?Date,
   isFullyBooked: boolean,
   hasNotFullyBookedTimes: boolean,
   description: ?HTMLString,
   format: ?EventFormat,
   bookingType: ?string,
-  image: ?Picture,
+  image: ?Image,
   interpretations: Array<Interpretation>,
   eventbriteId?: ?string,
+  audience: Audience,
+  series: EventSeries[],
+  schedule?: Array<Event>,
   // These are used for when we have a human written string for the dates.
   // Shouldn't really happen, but we have manually added promos at the moment.
   // Hence the nullable key - easier than implementing schedules for 1 event.
   dateString?: ?string,
-  timeString?: ?string
+  timeString?: ?string,
```

Comment:
what's with the trailing comma?

Reply:
[Everyone else is doing it](http://docs.scala-lang.org/sips/completed/trailing-commas.html)

Reply:
Ah fair enough. If it's enforceable by the linter that'd also be nice.